src/main/java/com/ubc/cpsc304/hotelify/service/ReviewService.java

```diff
@@ -0,0 +1,32 @@
+package com.ubc.cpsc304.hotelify.service;
+
+import com.ubc.cpsc304.hotelify.controller.dto.ReviewRequestDto;
+import com.ubc.cpsc304.hotelify.entity.Review;
+import com.ubc.cpsc304.hotelify.repository.ReviewRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewService {
+    private final ReviewRepository reviewRepository;
+
+    public ReviewService(ReviewRepository reviewRepository) {this.reviewRepository = reviewRepository;}
+
+    public Review createReview(ReviewRequestDto reviewRequestDto){
+        Long review_id = reviewRequestDto.getReview_id();
+
+        if (this.reviewRepository.findById(review_id).isPresent()) {
```

Comment:
you dont need this logic since we will never duplicate a review because of its id is auto-incremented from 1.